registry/spec.md

```diff
@@ -987,19 +1081,21 @@ Content-Length: nnnn
 **Example:**
 
 Request:
-```
+
+``` meta
 TODO
 ```
 
-#### Updating a Resource
+##### Updating a Resource
 
 This will update the latest version of a Resource. Missing Registry HTTP
 headers MUST NOT be interpreted as deleting the attribute. However, a Registry
 HTTP headers with an empty string for its value MUST be interpreted as a
 request to delete the attribute.
 
 The request MUST be of the form:
-```
+
+``` meta
 PUT /GROUPs/ID/RESOURCEs/ID[?epoch=EPOCH]
```

Comment:
For a follow-on PR... but I'm wondering if we should move `epoch` to be an HTTP like all other ?meta properties.